src/mapml/layers/TemplatedImageLayer.js

```diff
@@ -45,6 +47,7 @@ export var TemplatedImageLayer =  L.Layer.extend({
             this._imageOverlay._overlayToRemove = overlayToRemove._url;
             this._imageOverlay.on('load error', function () {map.removeLayer(overlayToRemove);});
         }
+        this.location = loc;
```

Comment:
What's the use for `this.location`? I can see it is only used in the tests. I don't think it should be added if it's only use is for testing?

Reply:
Yes it is only for testing. if it is not appropriate to add it, I can try to find another way to test it. I just tried some ways to test (count the  number of layer etc.), and then found they are much harder than testing in this way